### PR TITLE
improve instructions and divide between dev and translator

### PR DESCRIPTION
**Pull request by ann-kilzer**
What do you think about these? I try not to go too heavy with the JSON lesson, but would at least like to mention what they're looking at.

### CODE REVIEW

**Comment by theyokohamalife** [comment]
Let's use the findadoc.jp domain. Other than that it looks great!

File: README.md [hunk 1]
```diff
@@ -1,11 +1,18 @@
 # Find a Doc - Localization
 
-Find a Doc is a free online resource aimed at helping connect the foreign community in Japan with health services in their native language.
+[Find a Doc](https://findadocjp.org/) is a free online resource aimed at helping connect the foreign community in Japan with health services in their native language.
```
Comment (theyokohamalife) [inline README.md:3]: Update to findadoc.jp

File: README.md [hunk 2]
```diff
@@ -15,11 +22,23 @@ If you're new to Git, here are some easy steps you can take to get started:
 1. Go into the [locales](https://github.com/ourjapanlife/findadoc-localization/tree/main/locales) folder and click on the language file you wish to contribute to. It will take you to a screen that will show you its content. (Note if you are creating a new language file, copy from `en.json` and please name the new file using the two-letter language codes defined [here](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes))
    ![Select Language](./images/01.png)
 
-2. Click the `edit` button so you can directly edit the translation through your browser. **Use the `en.json` file as the reference content.**
+2. Click the `edit` button so you can directly edit the translation through your browser. **Use the `en.json` file and the [website](https://findadocjp.org/) as the reference content.**
```
Comment (theyokohamalife) [inline README.md:25]: findadoc.jp here as well

**Review by theyokohamalife — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ann-kilzer) [reply README.md:3]: Can you add it and merge on my behalf? Otherwise I can get to it late tonight

**Review by theyokohamalife — APPROVED** [review]
LGTM!

**Pull request merged**